### DISCUSSION TITLE
Why `]d` doesn't add to the jumplist?

### DISCUSSION THREAD

**Discussion by sijow** [helix-editor/helix#10475] [Q&A]
What's the logic behind having "Goto next diagnostic" not add to the jumplist? To me it looks very similar to other kinds of jumps that do change the jumplist.

**Comment by YaLTeR** [comment]
I had the same question. I noticed that this was one of the jumplist differences compared to neovim where goto prev/next diagnostic adds the location to the jumplist. Same for other goto prev/next commands like goto next git change.
